BUILD

```diff
@@ -67,14 +67,28 @@ container_bundle(
         "gcr.io/{PROJECT_ID}/python3:debug-nonroot": "//experimental/python3:debug_nonroot",
         "gcr.io/{PROJECT_ID}/python2.7:latest": "//experimental/python2.7:python27",
         "gcr.io/{PROJECT_ID}/python2.7:debug": "//experimental/python2.7:debug",
-        "gcr.io/{PROJECT_ID}/nodejs:latest": "//experimental/nodejs:nodejs10",
-        "gcr.io/{PROJECT_ID}/nodejs:10": "//experimental/nodejs:nodejs10",
-        "gcr.io/{PROJECT_ID}/nodejs:12": "//experimental/nodejs:nodejs12",
-        "gcr.io/{PROJECT_ID}/nodejs:14": "//experimental/nodejs:nodejs14",
-        "gcr.io/{PROJECT_ID}/nodejs:debug": "//experimental/nodejs:nodejs10_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:10-debug": "//experimental/nodejs:nodejs10_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:12-debug": "//experimental/nodejs:nodejs12_debug",
-        "gcr.io/{PROJECT_ID}/nodejs:14-debug": "//experimental/nodejs:nodejs14_debug",
+        "gcr.io/{PROJECT_ID}/nodejs:latest": "//experimental/nodejs:nodejs10_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:10": "//experimental/nodejs:nodejs10_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:12": "//experimental/nodejs:nodejs12_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:14": "//experimental/nodejs:nodejs14_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:10-debug": "//experimental/nodejs:nodejs10_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:12-debug": "//experimental/nodejs:nodejs12_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs:14-debug": "//experimental/nodejs:nodejs14_debug_debian9",
+        "gcr.io/{PROJECT_ID}/nodejs-debian9:10": "//experimental/nodejs:nodejs10_debian9",
```

Comment:
We should have `gcr.io/{PROJECT_ID}/nodejs-debian9:latest` too.

Reply:
done